packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -369,80 +375,44 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 							</span>
 						</p>
 						<div>
-							<FocusedLaunchSummaryItem
-								isLoading={ ! defaultFreePlan || ! defaultPaidPlan }
-								onClick={ () => defaultPaidPlan && setPlan( defaultPaidPlan ) }
-								isSelected={ isPlanSelected( defaultPaidPlan ) }
-							>
-								<LeadingContentSide
-									label={
-										/* translators: %s is WordPress.com plan name (eg: Premium Plan) */
-										sprintf( __( '%s Plan', __i18n_text_domain__ ), defaultPaidPlan?.title ?? '' )
-									}
-									badgeText={
-										defaultPaidPlan?.isPopular ? __( 'Popular', __i18n_text_domain__ ) : ''
-									}
-								/>
-								<TrailingContentSide nodeType="PRICE">
-									<span>{ defaultPaidPlan && planPrices[ defaultPaidPlan?.storeSlug ] }</span>
-									<span>
-										{
-											// translators: /mo is short for "per-month"
-											__( '/mo', __i18n_text_domain__ )
-										}
-									</span>
-								</TrailingContentSide>
-							</FocusedLaunchSummaryItem>
-
-							{ nonDefaultPaidPlan && (
-								<FocusedLaunchSummaryItem
-									onClick={ () => setPlan( nonDefaultPaidPlan ) }
-									isSelected={ isPlanSelected( nonDefaultPaidPlan ) }
-								>
-									<LeadingContentSide
-										label={
-											/* translators: %s is WordPress.com plan name (eg: Premium Plan) */
-											sprintf(
-												__( '%s Plan', __i18n_text_domain__ ),
-												nonDefaultPaidPlan.title ?? ''
-											)
-										}
-										badgeText={
-											nonDefaultPaidPlan.isPopular ? __( 'Popular', __i18n_text_domain__ ) : ''
-										}
-									/>
-									<TrailingContentSide nodeType="PRICE">
-										<span>{ planPrices[ nonDefaultPaidPlan?.storeSlug ] }</span>
-										<span>
-											{
-												// translators: /mo is short for "per-month"
-												__( '/mo', __i18n_text_domain__ )
-											}
-										</span>
-									</TrailingContentSide>
-								</FocusedLaunchSummaryItem>
+							{ allAvailablePlans.map(
+								( plan ) =>
+									plan && (
+										<FocusedLaunchSummaryItem
+											isLoading={ ! defaultFreePlan || ! defaultPaidPlan }
+											onClick={ () => plan && setPlan( plan ) }
+											isSelected={ plan && isPlanSelected( plan ) }
```

Comment:
We know that `plan` is not null at this point.

Reply:
You're absolutely right and the check on `plan` is redundant, hence I removed it (and the other's as well).